client/public/assets/css/bootstrap.paper.css

```diff
@@ -6335,7 +6335,7 @@ button.close {
   clear: both;
 }
 .center-block {
-  display: block;
+  display: inline-block;
```

Comment:
We're not using `bootstrap.paper.css` in production. Any reason why you changed this? If we don't need this change, let's revert the file.

If you do need `inline-block`, then I would recommend doing it in `app.css` instead of modifying the bootstrap styles.